subprojects/libportal.wrap

```diff
@@ -0,0 +1,4 @@
+[wrap-git]
+directory=libportal
+url=https://github.com/jsparber/libportal.git
```

Comment:
Is this correct? if so it should be noted in the commit message at least that we're not using upstream.

Reply:
Yes, we currently require this version because we merged the notifications portal v2 changes which depend on a libportal branch/PR to add support for the notifications portal v2 API.

This is an unfortunate circular dependency we should drop in the longer term by moving the tests to the pytest.

Reply:
the unfortunate part is that the **build** has been broken since and the proper solution instead is

1. disable said test
or
2. implement the missing function directly in the test.

Reply:
I don't understand. With this PR, the local and CI builds both use libportal as subproject which implements all the functions required by the tests.

Reply:
using a fork like that is NOT the proper fix. If you are blocked by some PR being merged elsewhere then the QUICK solution is the one I gave above, that will UNBLOCK other contributors. I still don't understand how CI didn't break (tbf I didn't look at the problem) but then that mean the CI workflow has a bug there too.

Reply:
See https://github.com/flatpak/xdg-desktop-portal/pull/1484

Reply:
Sorry, but I don't see what's wrong with specifying which libportal commit is required to run the tests. Disabling some tests on the other hand just makes our testing worse while also not making it clear which libportal version is required and potentially having different versions in CI and on development setups.

Reply:
Changed it to build libportal from upstream main.

Reply:
I'm guessing we're talking about https://github.com/flatpak/libportal/pull/147/?

Best to check in meson if `xdp_portal_get_supported_options` exists and ifdef those out